app/src/main/java/io/github/droidkaigi/confsched2018/presentation/speaker/SpeakerDetailActivity.kt

```diff
@@ -48,12 +48,22 @@ class SpeakerDetailActivity : AppCompatActivity(), HasSupportFragmentInjector {
         }
     }
 
+    override fun onDestroy() {
+        currentSpeakerId = null
+        super.onDestroy()
+    }
+
     companion object {
         const val EXTRA_SPEAKER_ID = "EXTRA_SPEAKER_ID"
+        private var currentSpeakerId: String? = null
```

Comment:
I think it's not good to save `currentSpeakerId` in companion object.
How about checking `speakerDetailViewModel.speakerId` here instead?
https://github.com/DroidKaigi/conference-app-2018/blob/6e0afdf27387230cd70e35c9fc038733d023d58a/app/src/main/java/io/github/droidkaigi/confsched2018/presentation/speaker/SpeakerDetailFragment.kt#L78

What do you think?

Reply:
ClickListener is in SpeakersSummaryLayout, so I make a commit to let it do nothing in SpeakerDetailFragment.